io.github.davejarvis.kmcaster.yaml

```diff
@@ -0,0 +1,37 @@
+id: io.github.davejarvis.kmcaster
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+command: /app/bin/run.sh
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin
+  - --socket=x11
```

Comment:
you should also add --share=ipc if you're using x11 to get high performance graphics